reliontomo/__init__.py

```diff
@@ -32,7 +34,13 @@
 
 class Plugin(relion.Plugin):
     _supportedVersions = [V3_0]
+    _homeVar = RELIONTOMO_HOME
 
     @classmethod
     def defineBinaries(cls, env):
         pass
+
+    @classmethod
+    def IS_30(cls):
+        environ = pwutils.Environ(os.environ)
+        return 'relion-3.0' in environ[RELIONTOMO_HOME]
```

Comment:
I think, this var could be defined in the _defineVariables method of Plugin class with a default value to 3.0 and then use cls.getHome() instead of Environ.